docs/setup/facilities_locator.md

```diff
@@ -4,3 +4,31 @@ For the current maps.va.gov endpoint, you will need to add the VA internal root
 CA certificate to your trusted certificates. With `homebrew` this is typically
 done by appending the exported/downloaded certificate to
 `<HOMEBREW_DIR>/etc/openssl/cert.pem`.
+
+#### SQL52
+
+When running facilities sidekiq jobs, mental health phone numbers depend on
+a database called SQL52 that is behind the VA network. To get around this
+locally you can run your own dockerized SQL52 container locally. The following
+instructions will help you run SQL52 locally:
+
+1. `docker pull dsva/vets-api-sql52-stubbed:latest`
+1. `docker run -e ACCEPT_EULA=Y -e SA_PASSWORD=password1! -p 1433:1433 -d dsva/vets-api-sql52-stubbed:latest`
+1. In `config/settings.local.yml`, you will also need to set these variables
+
```

Comment:
We may need to overhaul this, considering the code doesn't use a SQL52 client at all anymore